src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -45,7 +45,9 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Remote.ServiceHub" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Remote.Workspaces" />
     <RestrictedInternalsVisibleTo Include="Microsoft.CodeAnalysis.LiveUnitTesting.Orchestrator" Partner="UnitTesting" />
+    <RestrictedInternalsVisibleTo Include="Microsoft.CodeAnalysis.LiveUnitTesting.Orchestrator" Partner="UnitTesting" Key="$(UnitTestingKey)" />
```

Comment:
tag @mavasani to see if the banned api analyzer would work correctly for this case?

Reply:
The cleanest way to keep things from breaking is preserving the key during the source code move (i.e. no Roslyn change required). If the key does change, it _might_ be possible to have duplicate lines here (one with each key), but if that doesn't work it will need to be a coordinated insertion.

Reply:
It wouldn't be ideal to preserve the existing key, and having duplicate lines here does not work in my testing. Whether it's two `RestrictedInternalsVisibleTo` or an additional `InternalsVisibleTo`. So it looks like it will have to be a coordinated insertion if there isn't another way.
What would be the process of doing so?

Reply:
I was incorrect in my testing. Older packages were being pulled in from my .nuget cache and producing unexpected results since the built dev package versions didn't change.
Having duplicate roslyn versions do work, and so this can be phased in. So, pushing this change in shouldn't break the current testimpact, and should also work with the new binaries with different public keys.

Reply:
Do we have a work item tracking removal of these duplicate entries?

Reply:
I have created one here https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1155201